core/templates/dev/head/css/oppia.css

```diff
@@ -65,6 +65,10 @@ button.md-button.md-default-theme.oppia-learner-continue-button:hover {
   background-color: #115FD4;
 }
 
+/* Attributes for material-icons. */
+.material-icons{
```

Comment:
Careful. With this you are affecting everything with .material-icons on the site, not just the ones you changed. Is this intentional?